docs/content/en/integrations/api-v2-docs.md

```diff
@@ -171,3 +173,39 @@ Example for importing a scan result:
        -   Browse for the file to upload
 
 -   Click send
+
+Clients / API Wrappers
+----------------------
+
+| Wrapper                      | Notes                   |
+| -----------------------------| ------------------------|
+| [Specific python wrapper](#openapi-python-wrapper)      | working (2021-01-21)    |
+| [Openapi python wrapper](#openapi-python-wrapper)       | proof of concept only   |
+| [Java library](#java-library)                 | working (2021-08-30)    |
+| [Image using the Java library](#image-using-the-java-library) | working (2021-08-30)    |
+| [.Net/C# library](#netc-library)              | working (2021-06-08)    |
+| [dd-import](#dd-import)                    | working (2021-08-24)    |
+
+### Specific python wrapper
+* Install the DefectDojo Python API via `pip install defectdojo_api`(outdated) or clone
+  the [repository](https://github.com/DefectDojo/defectdojo_api).
+* Browse the API on
+  [SwaggerHub](https://app.swaggerhub.com/apis/DefectDojo/defect-dojo_api_v_2/1.0.0).
+  [![Swagger Status](http://online.swagger.io/validator?url=https://api.swaggerhub.com/apis/DefectDojo/defect-dojo_api_v_2/1.0.0)](https://app.swaggerhub.com/apis/DefectDojo/defect-dojo_api_v_2/1.0.0)
+
+### Openapi python wrapper
```

Comment:
I think this is duplicated. We should decide if we want the table or the sections.

Reply:
How does this work ‽